### DISCUSSION TITLE
.NET Core Package Groups

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#7381] [Questions]
I found https://github.com/wixtoolset/issues/issues/6099 that indicated .NET Core 3.1 packages were done in wix.4.0-preview.0.  However the PR seems to be to the old standalone NetFx repos.  When I search through the Wix4 repos I only find reference in some test data.  I don't see any .NET Core PackageGroups in the wixlibs.

I know .NET 6/7 has been revving pretty hard... like monthly.  Do we want someone to do this work or is the idea just let it go and people can publish  their apps not framework dependent?  For my purposes I think it would be good to have atleast one version of the hosting bundle per major release.  

Also, are there any good resources on learning how to write my own wixext?  I'm thinking I'd like to make a collection of package groups and make them available for others to use.

**Comment by rseanhall** [comment]
There were several factors to pulling the .NET Core package groups, but discouraging framework dependent apps was not one of them.

1. We have found that there are a lot of people that don't use the built-in package groups but copy our source code and modify it for their use case.
2. We don't have the bandwidth to release multiple times a year but .NET Core is releasing every month. Their support policy is that only the latest release is supported. We were uncomfortable providing packages that would be out of support before we even shipped.
3. Let's just say the core maintainers were not happy with how some members of this community interacted with us with the built-in .NET Framework package groups.

I don't know of any resources on how to write a v4 wixext other than looking at the source code for ones provided by WiX.
